### PR TITLE
Naming consistency of API mutations

### LINKED ISSUE #2757: Make API field names more consistent
We should introduce some project-wide pattern for endpoints and mutations naming.
In some parts of the project, we have different styles of naming for mutations and it makes hard to figure out if some API scenarios are covered or not.

I recommend to keep following approach:
`<Entity><Action` examples:
`categoryCreate`
`orderUpdate`

now we have for example `orderAddNote` and it should be refactored to `orderNoteAdd`
Also, we should consider using the same verbs for common operations like `Create`, `Update` or `Delete`. Now it's mixed up sometimes.

The second recommendation is to sort alphabetically and group mutations in `api.py` file because now it's hard to find interesting mutations there.

What do you think?

### PR DESCRIPTION
**Pull request by the-bionic**
I want to merge this change because...
resolves #2757

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] GraphQL schema and type definitions are up to date.

### CODE REVIEW

File: saleor/graphql/api.py [hunk 1]
```diff
@@ -313,21 +314,14 @@ class Mutations(graphene.ObjectType):
...
-    draft_order_line_create = DraftOrderLineCreate.Field()
-    draft_order_line_delete = DraftOrderLineDelete.Field()
-    draft_order_line_update = DraftOrderLineUpdate.Field()
-    draft_order_update = DraftOrderUpdate.Field()
     order_fulfillment_cancel = FulfillmentCancel.Field()
     order_fulfillment_create = FulfillmentCreate.Field()
-    order_fulfillment_update_tracking = FulfillmentUpdateTracking.Field()
-    order_add_note = OrderAddNote.Field()
+    order_fulfillment_update_tracking = FulfillmentTrackingUpdate.Field()
+    order_add_note = OrderNoteAdd.Field()
```
Comment (NyanKiyoshi) [inline saleor/graphql/api.py]: We have order_**add_note** but then we have **NoteAdd**.  We also have other cases below where we also have *ObjectName***Action** and not **Action**ObjectName. Plus, "OrderAddNote" sounds more correct English.

Comment (the-bionic) [reply saleor/graphql/api.py]: I think ```CreateOrder``` sounds more correct than ```OrderCreate```, but that is the convention. Same applies to OrderNoteAdd. 

Should OrderNoteCreate be used here instead?

I will update the variable names, must have missed that.

Comment (the-bionic) [reply saleor/graphql/api.py]: I think ```CreateOrder``` sounds more correct than ```OrderCreate```, but that is the convention. Same applies to OrderNoteAdd. 

Should OrderNoteCreate be used here instead?

I will update the variable names, must have missed that.

File: saleor/graphql/api.py [hunk 2]
```diff
@@ -269,14 +269,15 @@ def resolve_address_validator(self, info, input):
...
 class Mutations(graphene.ObjectType):
-    assign_navigation = AssignNavigation.Field()
+    address_create = AddressCreate.Field()
+    address_update = AddressUpdate.Field()
+    address_delete = AddressDelete.Field()
 
-    token_create = CreateToken.Field()
-    token_refresh = graphql_jwt.Refresh.Field()
-    token_verify = VerifyToken.Field()
+    assign_navigation = AssignNavigation.Field()
```
Comment (maarcingebala) [inline saleor/graphql/api.py]: This should be renamed to `navigation_assign`, according to the <enitity><action> rule.

File: saleor/graphql/api.py [hunk 3]
```diff
@@ -286,24 +287,24 @@ class Mutations(graphene.ObjectType):
     category_delete = CategoryDelete.Field()
     category_update = CategoryUpdate.Field()
 
+    collection_create = CollectionCreate.Field()
+    collection_update = CollectionUpdate.Field()
+    collection_delete = CollectionDelete.Field()
+    collection_products_add = CollectionProductsAdd.Field()
```
Comment (maarcingebala) [inline saleor/graphql/api.py]: As suggested [in this comment](https://github.com/mirumee/saleor/issues/2757#issuecomment-426213619) I would consider "collection" to be the entity. So the original name was correct.

File: saleor/graphql/api.py [hunk 4]
```diff
@@ -377,6 +359,25 @@ class Mutations(graphene.ObjectType):
...
 
+    shop_domain_update = ShopDomainUpdate.Field()
+    shop_settings_update = ShopSettingsUpdate.Field()
+    homepage_collection_update = HomepageCollectionUpdate.Field()
+
+    staff_create = StaffCreate.Field()
+    staff_update = StaffUpdate.Field()
+    staff_delete = StaffDelete.Field()
+
+    set_password = SetPassword.Field()
```
Comment (maarcingebala) [inline saleor/graphql/api.py]: `password_set`

**Comment by maarcingebala** [comment]
@the-bionic Changes look good, but we would also have to regenerate GraphQL schema (`npm run build-schema`) and ensure that the build of frontend code isn't failing after that.

**Comment by maarcingebala** [comment]
@the-bionic There has been a lot of changes in the API recently and this PR is not up-to-date with them. Will you have to take a look at it in the nearest future? I just need to know whether I can expect some updates from your side. If not, maybe I'll get someone from our team to take this issue.

**Pull request closed without merging**